src/vcProject.cpp

```diff
@@ -124,20 +124,39 @@ udError   vcProject_CreateFileScene(vcState *pProgramState, const char *pFileNam
   if (pProjectName == nullptr || pProjectName[0] == '\0')
     return udE_InvalidParameter;
 
-  if (udFileExists(pFileName) == udR_Success)
-    return udE_WriteFailure;
-
-  udProject *pNewProject = nullptr;
-  udError error = udProject_CreateInFile(&pNewProject, pProjectName, pFileName);
-  if (error != udE_Success)
-    return error;
+  udError error = udE_Success;
+  udProject* pNewProject = nullptr;
+  bool overrideProject = false;
 
+  if (udFileExists(pFileName) == udR_Success)
+  {
+    overrideProject = true;
+    error = udProject_CreateInMemory(&pNewProject, pProjectName);
```

Comment:
This seems wrong? @OsiJr does `udProject_CreateInFile` continue to write to disk as changes are made?

Reply:
@samsinsane Would you prefer that I use udProject_CreateInFile with a temp file and once saved to the file provided, delete the temp file ?

Reply:
If my understanding is correct, `udProject_CreateInFile` will write any future changes to the file provided. The correct thing would be to do the detection immediately, and handle it before we ever get to a `udProject_*` function - assuming my understanding is correct.